formula-test/src/main/java/com/instacart/formula/test/TestFormula.kt

```diff
@@ -3,77 +3,81 @@ package com.instacart.formula.test
 import com.instacart.formula.Action
 import com.instacart.formula.Evaluation
 import com.instacart.formula.Formula
+import com.instacart.formula.IFormula
 import com.instacart.formula.Snapshot
+import com.instacart.formula.test.utils.RunningInstanceManager
 import java.util.concurrent.atomic.AtomicLong
 
 /**
- * Test formula is used to provide a fake formula implementation. It allows you to [send][output]
- * output updates and [inspect/interact][input] with input.
+ * Test formula is used to replace a real formula with a fake implementation. This allows you to:
+ * - Verify that parent passes correct inputs. Take a look at [input].
+ * - Verify that parent deals with output changes correctly. Take a look at [output]
```

Comment:
Could we also call out that we support the running count? This is a handy feature that I think a lot of devs don't know about.